modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/portlet/PortletInit.es.js

```diff
@@ -838,7 +838,7 @@ class PortletInit {
 			);
 		}
 
-		if (!isDefAndNotNull(handle)) {
+		if (handle === undefined || handle === null) {
```

Comment:
Just as an FYI, when we [turn on the `eqeqeq` lint rule](https://github.com/liferay/liferay-frontend-projects/issues/14) we'll probably want to standardize on `handle == null` to replace `handle === undefined || handle === null` — it's basically the only place we'll allow `==`, as a hat-tip to idiomatic JS and it's special snowflake status as a language that has both `undefined` and `null`.